### PR TITLE
feat: #23 auro-pane selected attribute

### LINKED ISSUE #23: auro-pane: selected attribute reflection
## Describe the bug
In a Svelte application, I cannot select the pane by querying the 'select' attribute, because the property is not reflected. I need to get the element's position so that I can center the pane when it is in a carousel.

## Expected behavior
The 'selected' attribute is rendered on the element.

## Relevant Framework 
- Svelte

### PR DESCRIPTION
**Pull request by ftaylor-alaska**
Fixes #23

### CODE REVIEW

File: src/auro-pane.js [hunk 1]
```diff
@@ -43,9 +43,9 @@ class AuroPane extends LitElement {
         attribute: "aria-hidden"
       },
       date: { type: String },
-      disabled: { type: Boolean },
+      disabled: { type: Boolean, reflect: true },
```
Comment (jason-capsule42) [inline src/auro-pane.js:46]:
```suggestion
      disabled: {
        type: Boolean,
        reflect: true
      },
```

File: src/auro-pane.js [hunk 2]
```diff
@@ -43,9 +43,9 @@ class AuroPane extends LitElement {
...
       price: { type: String },
-      selected: { type: Boolean },
+      selected: { type: Boolean, reflect: true },
```
Comment (jason-capsule42) [inline src/auro-pane.js:48]:
```suggestion
      disabled: {
        type: Boolean,
        reflect: true
      },
```

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]
The build linter rules will fail if the object keys are not on separate lines.

**Comment by blackfalcon** [comment]
@ftaylor-alaska, per the comment from @geoffrich  this is a `feature` not a `bug`. Please update the commit message. Please review the Contributing guidelines https://auro.alaskaair.com/contributing

Please update the commit message to speak to the scope and the update, e.g. 

```
feat(selected attr): update to reflect true #23
```

An additional note, it is helpful to allow for feedback on the issue itself prior to creating and submitting the PR. This allows the Auro team to provide any feedback necessary prior to work being done and help the PR be released as soon as possible. We appreciate the support.

**Comment by blackfalcon** [comment]
Question: is there a process you are using outside of Github? We are curious as to why pre-commit hooks are not catching issues also, how did you create this PR? Why was all the template content deleted?

**Pull request closed without merging**